source/crud/tests/v2/find-allowdiskuse.yml

```diff
@@ -0,0 +1,50 @@
+runOn:
```

Comment:
Until someone writes a POC we won't be sure these tests are correct.

Reply:
The server work is done and available on "latest" so I think it makes sense for you to do it in the .NET driver.  It seems straightforward.

Reply:
Will do

Reply:
Done.

https://github.com/rstam/mongo-csharp-driver/pull/141